docs/internal/internal-user-interface.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Internal User Interface
+---
+
+# Internal User Interface
+
+Our internal dashboard is primarily ERB views that render on the server.
+Largely, we try to adhere to
+[ActionView](https://guides.rubyonrails.org/action_view_overview.html)'s
+conventions in these views.
+
+For layout, basic styles, and some interactions, we use
+[Bootstrap 4](https://getbootstrap.com/). DEV isn't dedicated to using Bootstrap
+for everything, but because our design team hasn't spent much time on these
+views, we find it's easier to stick with something many developers already know.
+
+When a view requires some custom interactivity, we've historically leaned on
+vanilla JavaScript or jQuery, but going forward we've elected to use
+[StimulusJS](https://stimulusjs.org/) for DOM manipulation and interactivity
+inside of internal.
+
+# Forms
+
+Inside of the internal views, we're
+[actively moving from the old ERB syntax for forms](https://m.patrikonrails.com/rails-5-1s-form-with-vs-old-form-helpers-3a5f72a8c78a).
+We tend to prefer the `form_with` helper over the previous `form_for` and
+`form_tag` helpers.
+
+# StimulusJS
+
+Stimulus is a modest frontend framework, and its primary purpose is manipulating
+HTML. It doesn't contain a templating language akin to JSX. Think of it as a
+more scalable version of jQuery.
```

Comment:
I understand why but I don't think we should define Stimulus in relation to React or jQuery. If someone tells me Stimulus is a version of jQuery I think of jQuery utilities and all the rendering tricks, not that it's better because of code organization and structure. 

I'd directly quote the homepage: "Stimulus is a JavaScript framework with modest ambitions. It doesn’t seek to take over your entire front-end—in fact, it’s not concerned with rendering HTML at all." :)

Reply:
Also, JSX is not templating, it's more of a DSL.